crates/proto/proto/lsp.proto

```diff
@@ -543,6 +543,14 @@ message UpdateLanguageServer {
     }
 }
 
+message ShowDocument {
+    uint64 project_id = 1;
+    string uri = 2;
+    bool take_focus = 3;
+    optional PointUtf16 selection_start = 4;
+    optional PointUtf16 selection_end = 5;
```

Comment:
I feel like `take_focus` and `selection_start`/`selection_end` don't make sense in RPC calls, as they can only be handled (reasonably) by the requesting side.

We should prolly return buffer_id from `ShowDocument` RPC call.